### DISCUSSION TITLE
What is the event for when you "like" a message?

### DISCUSSION THREAD

**Discussion by chneau** [yagop/node-telegram-bot-api#1195] [Q&A] [answered]
Hello,

I'm writing a bot connected to an AI Api. I would like to have an event like `bot.on("liked_message", (event:?) => {...}`.
I'm reading the doc here https://github.com/yagop/node-telegram-bot-api/blob/master/doc/usage.md but can't find one that is doing it.
Does someone know which one is it?

Closest thing I could read from the documentation is https://core.telegram.org/bots/api#setmessagereaction 

Kind regards,
Charles

**Comment by danielperez9430** [comment, marked as answer]
Telegram Updates Events: https://core.telegram.org/bots/api#update

<img width="822" alt="image" src="https://github.com/yagop/node-telegram-bot-api/assets/10388515/f262ddb3-08d6-42b6-b3c6-1018eb6a70fd">

You need to listen the "message_reaction" event.

Example of use:

```js
const TelegramBot = require('node-telegram-bot-api');

const bot = new TelegramBot(TOKEN, {
    polling: {
        params: {
            "allowed_updates": JSON.stringify(["update_id", "message", "edited_message", "channel_post", "edited_channel_post", "inline_query", "chosen_inline_result", "callback_query", "shipping_query", "pre_checkout_query", "poll", "poll_answer", "my_chat_member", "chat_member", "message_reaction"])
        }
    }
});

bot.on('message_reaction', (msg) => {
    const chatId = msg.chat.id;
    console.log(msg);
    bot.sendMessage(chatId, 'Received your reaction message');
});

```
